lua/neotest-plenary/base.lua

```diff
@@ -6,9 +6,17 @@ function M.is_test_file(file_path)
   return vim.endswith(file_path, "_spec.lua")
 end
 
-function M.get_strategy_config(strategy, python, python_script, args)
+function M.get_strategy_config(strategy)
   local config = {
-    dap = nil, -- TODO: Add dap config
+    dap = function()
+      return vim.tbl_extend("keep", {
+        type = "nlua",
+        request = "attach",
+        name = "Neotest Debugger",
+        host = "127.0.0.1",
```

Comment:
Both, host and ports should be part of the configuration. Ideally, they should be provided by the following dependency: [one-small-step-for-vimkind](https://github.com/jbyuki/one-small-step-for-vimkind/blob/f239ca0dc772f214e7ff0627ccbd45bceae88bf8/lua/osv/init.lua#L1092-L1101)